src/internal.rs

```diff
@@ -879,9 +930,13 @@ pub(crate) struct Rav1dFrameData {
 impl Rav1dFrameData {
     pub unsafe fn zeroed() -> Self {
         let mut data: MaybeUninit<Rav1dFrameData> = MaybeUninit::zeroed();
+        addr_of_mut!((*data.as_mut_ptr()).refp).write(Default::default());
+        addr_of_mut!((*data.as_mut_ptr()).cur).write(Default::default());
+        addr_of_mut!((*data.as_mut_ptr()).sr_cur).write(Default::default());
         addr_of_mut!((*data.as_mut_ptr()).out_cdf).write(Default::default());
         addr_of_mut!((*data.as_mut_ptr()).tiles).write(vec![]);
         addr_of_mut!((*data.as_mut_ptr()).a).write(vec![]);
+        addr_of_mut!((*data.as_mut_ptr()).rf).write(Default::default());
         addr_of_mut!((*data.as_mut_ptr()).lowest_pixel_mem).write(Default::default());
         addr_of_mut!((*data.as_mut_ptr()).frame_thread).write(Default::default());
         addr_of_mut!((*data.as_mut_ptr()).lf).write(Default::default());
```

Comment:
I'm not sure which PR added this, but this is the exact opposite of how we should be initializing things.  If done this way, we can trivially forget to initialize a field.  If we initialize directly, we can't forget a field.  The remaining individual fields should be `mem::zeroed()`.

Reply:
Sure, but I thought you were doing that initialization change so I didn't want to alter anything substantially here.

Reply:
Okay then that's fine. I can do it.